src/lib/logical_query_plan/export_node.cpp

```diff
@@ -6,34 +6,26 @@
 
 namespace hyrise {
 
-ExportNode::ExportNode(const std::string& init_table_name, const std::string& init_file_name,
-                       const FileType init_file_type)
-    : AbstractNonQueryNode(LQPNodeType::Export),
-      table_name(init_table_name),
-      file_name(init_file_name),
-      file_type(init_file_type) {}
+ExportNode::ExportNode(const std::string& init_file_name, const FileType init_file_type)
+    : AbstractNonQueryNode(LQPNodeType::Export), file_name(init_file_name), file_type(init_file_type) {}
 
 std::string ExportNode::description(const DescriptionMode mode) const {
-  std::ostringstream stream;
-  stream << "[Export] Name: '" << table_name << "'";
-  return stream.str();
+  return "[Export]";
```

Comment:
Comment only: I could add the export path here if desired an return sth like:
`[Export] to 'given/export/path.csv'`.
```suggestion
  return "[Export] to '" + file_name + '";
```
The same could be done for the operator description.

Reply:
Sounds good to me.

Reply:
Added the info. For the export operator, the description also prints the file type (binary/csv; `FileType::Auto` is only translated to an actual type in the operator).